cirq-ft/cirq_ft/_version.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Define version number here, read it from setup.py automatically"""
+
+__version__ = "1.2.0.dev"
```

Comment:
@verult Can we do `0.1.0dev` instead?

Reply:
A big benefit of keeping the same version is that the compatibility matrix between vendor packages and cirq-core is simple: vendor package version 1.2.3 depends on cirq-core 1.2.3. I think using 0.x.x for a new packages makes sense, and we can even bump major and minor versions at a different cadence as cirq-core if our release tools allow, but we should then document compatibility with cirq-core.

For example, if say cirq-ft 0.3.0 requires cirq-core 1.3 features:

* cirq-ft 0.1 - 0.2 : >= cirq-core 1.2
* cirq-ft 0.3: >= cirq-core 1.3

Reply:
IIUC the plan is to distribute cirq-ft together with cirq.  For example, the cirq metapackage currently includes all submodules including cirq-ft.

https://github.com/quantumlib/Cirq/blob/49b2cfb0a7568e4fd7e92cc563c8291a2a88b645/setup.py#L53-L54

If that is the case, I think it is better to keep cirq-ft version in sync with all other versions in cirq.  
We can consider cirq-ft to be a new capability provided with the next public version Cirq-1.2.0.